packages/orchestration/src/utils/start-helper.js

```diff
@@ -213,11 +222,13 @@ export const withOrchestration =
   (contractFn, opts) => async (zcf, privateArgs, baggage) => {
     const { marshaller, ...allOrchPowers } = privateArgs;
     const { storageNode: _, ...requiredOrchPowers } = allOrchPowers;
+    const { publishAccountInfo, ...restOpts } = opts ?? {};
     const { zone, ...tools } = provideOrchestration(
       zcf,
       baggage,
-      opts?.publishAccountInfo ? allOrchPowers : requiredOrchPowers,
+      publishAccountInfo ? allOrchPowers : requiredOrchPowers,
       marshaller,
+      restOpts,
```

Comment:
do you expect we'll need to thread more opts through? Passing through opaquely can lead to surprises.